hcloud/schema/floating_ip.go

```diff
@@ -91,3 +97,13 @@ type FloatingIPActionChangeDNSPtrRequest struct {
 type FloatingIPActionChangeDNSPtrResponse struct {
 	Action Action `json:"action"`
 }
+
+// FloatingIPChangeProtectionRequest defines the schema of the request to change the resource protection of a Floating IP.
+type FloatingIPChangeProtectionRequest struct {
```

Comment:
Should be `FloatingIPActionChangeProtectionRequest` to be consistent with other actions.